helm-chart-sources/logstream-leader/templates/_pod.tpl

```diff
@@ -93,6 +93,12 @@ containers:
       # Single Volume for persistence (CRIBL-3848)
       - name: CRIBL_VOLUME_DIR
         value: {{ .Values.config.criblHome }}/config-volume
+      # CPU Limit
+      - name: CRIBL_KUBE_CPU_LIMIT
```

Comment:
@pdugas, question
Kubernetes supports syntax such as '500m'
(see https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/#meaning-of-cpu).
I'm not sure if or how Kubernetes translates this syntax (if yes if this consistent behavior ?). If not, does this syntax would work as you expected, e.g. if 
CRIBL_KUBE_CPU_LIMIT would be set to 500m

Reply:
@michalbiesek The product already has code to handle this.  See that PR.